chutney/server/src/main/java/com/chutneytesting/execution/domain/PurgeServiceImpl.java

```diff
@@ -228,11 +229,13 @@ private PurgeExecutionService(
          */
         Set<Long> purgeExecutions() {
             Set<Long> deletedExecutionsIds = new HashSet<>();
+            LocalDateTime twentyFourHoursEarlier = LocalDateTime.now().minusHours(24);
```

Comment:
I prefer to have this filter defined statically with a meaningfull name and injected in the constructors (for scenario and campaign) with the **executionFilter** parameter.
Same for scenario, extract the existing filter in a meaningfull static value.